src/pages/collators/dialogs/DelegateToCollatorDialog.tsx

```diff
@@ -19,6 +19,7 @@ interface DelegateToCollatorDialogProps {
   tokenSymbol: string;
   visible: boolean;
   isDelegatingMore: boolean;
+  isDelegatingLess: boolean;
```

Comment:
We could also replace them with a `mode` so that we make it more explicit that they are mutually exclusive and are not supposed to be `true` both at the same time.

Reply:
Similar to the `useState` I described before

Reply:
Alright then, it'll be a bit more verbose but maybe more readable. Will fix it.